src/clj/rems/workflow/dynamic.clj

```diff
@@ -901,7 +1034,15 @@
         (is (= #{::approve ::reject ::return ::request-decision ::request-comment ::add-member ::invite-member}
                (possible-commands "assistant" submitted)))
         (is (= #{}
-               (possible-commands "somebody else" submitted))))
+               (possible-commands "somebody else" submitted)))
+        (testing "when there are invited members"
+          (is (contains? (possible-commands "applicant" (assoc submitted :invited-members [{:name "Some One" :email "some.one@example.org"}]))
+                         ::uninvite-member))
+          (is (contains? (possible-commands "assistant" (assoc submitted :invited-members [{:name "Some One" :email "some.one@example.org"}]))
+                         ::uninvite-member)))
+        (testing "when there are added members"
+          (is (contains? (possible-commands "applicant" (update submitted :members conj {:userid "someone"}))
+                         ::remove-member))))
```

Comment:
Please refactor this so that the application is produced from events with `apply-events`, similar to all the other tests; don't assoc stuff directly to `submitted`.

Reply:
Ok I based my tests on earlier code that didn't use the events.